config.toml

```diff
@@ -11,68 +11,11 @@
    # available in local disk
    StartInEpochEnabled = true
 
-   # SCDeployEnableEpoch represents the epoch when the deployment of smart contracts will be enabled
-   SCDeployEnableEpoch = 180
+   # ChainID identifies the blockChain
```

Comment:
could have been a more explanatory comment

Reply:
will be done on elrond-go master/development